src/languages/es.ts

```diff
@@ -4516,6 +4516,8 @@ const translations = {
                     benefit2: 'Reglas inteligentes de gastos',
                     benefit3: 'Flujos de aprobación de varios niveles',
                     benefit4: 'Controles de seguridad mejorados',
+                    toUpgrade: 'Para mejorar, haz clic',
+                    selectWorkspace: 'selecciona un espacio de trabajo y cambia el tipo de plan a Controlar.',
```

Comment:
@DylanDylann @carlosmiceli, in this copy, we mistakenly show the plan type(`Controlar`) twice. I will raise a follow-up PR, but before that, I’ve bumped it on Slack again to get confirmation from the team on this Spanish copy.